angel_system/data/medical/load_bbn_data.py

```diff
@@ -317,26 +338,41 @@ def bbn_activity_txt_to_csv(task, root_dir, output_dir, label_version):
         video_name = os.path.basename(video_dir)
         if video_name in KNOWN_BAD_VIDEOS:
             continue
-
+        
         print(action_txt_fn)
-        with open(action_txt_fn) as action_f:
-            lines = action_f.readlines()
-
+        # with open(action_txt_fn) as action_f:
+        #     lines = action_f.readlines()
+        
         # Create output csv
+        used_videos.append(video_name)
         csv_fn = f"{output_dir}/{video_name}_activity_labels_v{label_version}.csv"
         csv_f = open(csv_fn, "w")
         csv_f.write(
             "# 1: Detection or Track-id,2: Video or Image Identifier,3: Unique Frame Identifier,4-7: Img-bbox(TL_x,TL_y,BR_x,BR_y),8: Detection or Length Confidence,9: Target Length (0 or -1 if invalid),10-11+: Repeated Species,Confidence Pairs or Attributes\n"
         )
         csv_f.write('# metadata,fps: 1,"exported_by: ""dive:typescript"""\n')
-
-        for line in lines:
-            data = line.split("\t")
-
-            # Find frame filenames
-            start_frame = int(data[0])
-            end_frame = int(data[1])
-
+        
+        f = open(action_txt_fn, "r")
+        text = f.read()
+        f.close()
```

Comment:
A more robust version of this is:
```suggestion
        with open(action_txt_fn, "r") as f:
            text = f.read()
```
We generally don't want to be calling `.close()` on files and should leave that up to a context manager in case there is any errors during operation so that resource cleanup happens in due course.